packages/batch-submitter/test/batch-submitter/transaction-batch-submitter.spec.ts

```diff
@@ -193,13 +193,13 @@ describe('TransactionBatchSubmitter', () => {
       let receipt = await batchSubmitter.submitNextBatch()
       let logData = remove0x(receipt.logs[0].data)
       expect(parseInt(logData.slice(64 * 0, 64 * 1), 16)).to.equal(0) // _startingQueueIndex
-      expect(parseInt(logData.slice(64 * 1, 64 * 2), 16)).to.equal(5) // _numQueueElements
-      expect(parseInt(logData.slice(64 * 2, 64 * 3), 16)).to.equal(5) // _totalElements
```

Comment:
Had to update the tests here because I wasn't actually submitting all the blocks that I should have been cuz of the off-by-1